src/Keboola/DbExtractor/Extractor/Common.php

```diff
@@ -280,6 +286,23 @@ function ($table) {
         return array_values($tableDefs);
     }
 
+    public function getMaxOfIncrementalFetchingColumn(array $table): ?string
```

Comment:
just for future reference, this should be executed in some `runRetryableQuery` method.  We don't need it here because Common is just an example implementation though